Assets/MixedRealityToolkit.SDK/Inspectors/Dwell/DwellHandlerInspector.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell.Editor
+{
+    [CustomEditor(typeof(DwellHandler), true)]
+    public class DwellHandlerInspector : UnityEditor.Editor
```

Comment:
> I would recommend making these private classes, nested inside the class they are editing. This way, your `FindProperty` calls can use `nameof` which will make this code much more resilient to field renaming bugs. This would look something like:
> 
> ```cs
> partial class DwellHandler
> {
>     [CustomEditor(typeof(DwellHandler), true)]
>     private class DwellHandlerInspector : UnityEditor.Editor
>     {
>         ...
>         this.serializedObject.FindProperty(nameof(DwellHandler.dwellProfile));
>         ...
>     }
> }
> ```

I have not seen that pattern prevelant in MRTK. @wiwei , @keveleigh How do you folks feel about this ?

Reply:
The suggestion makes sense, though because we have prior art here and I think it would be best to keep consistent with the existing code.

As-is is fine because I think we'd value consistency in this case (despite being consistent with something that could obviously be improved) over the incremental value this provides

---
In reply to: [313542173](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5594#discussion_r313542173) [](ancestors = 313542173)